src/terminal/init.lua

```diff
@@ -18,6 +18,26 @@ local M = {
   _DESCRIPTION = "Cross platform terminal library for Lua (Windows/Unix/Mac)",
 }
 
+-- Helper function to resolve negative indices
+local function resolve_index(index, max_value)
+  if index < 0 then
+    return max_value + index + 1
+  end
+  return index
+end
+
+--- Returns the height of the terminal screen.
+-- @treturn number the height in rows
+function M.get_height()
+  return sys.get_terminal_size().height
+end
+
+--- Returns the width of the terminal screen.
+-- @treturn number the width in columns
+function M.get_width()
+  return sys.get_terminal_size().width
+end
+
```

Comment:
I do not like these functions. Making a function call to make a function call is inefficient.

Also, I think in most scenario's you'd need both height & width. If someone would use these instead, the inefficiency would be even bigger.